lib/connection.go

```diff
@@ -103,6 +107,11 @@ func (c *Conn) SetHosts(newhosts []string) {
 	c.initializeHostPool()
 }
 
+// Sets the timeout for each individual request.
+func (c *Conn) SetTimeout(timeout time.Duration) {
```

Comment:
Agreed, but the [places](https://github.com/launchdarkly/elastigo/pull/2/files#diff-82a1f084f78567b50e60c4977d1ed315R69) we actually use the httpClient aren't from methods on the `Conn` struct. I'll change it so it's a package function